lexibank_chenhmongmien.py

```diff
@@ -27,12 +27,19 @@ class HLanguage(Language):
     Location = attr.ib(default=None)
 
 
-class Dataset(NonSplittingDataset):
+class Dataset(MyDataset):
     dir = Path(__file__).parent
     id = "chenhmongmien"
     concept_class = HConcept
     language_class = HLanguage
 
+    form_spec = FormSpec(
```

Comment:
Important for all dataset that tinker with forms (i.e., most of them): do not overwrite, do not split yourself, just use the pre-defined parameters here. And use with care, in any case.